infrastructure/resource_group_level/main.bicep

```diff
@@ -137,9 +137,7 @@ resource containerRegistry 'Microsoft.ContainerRegistry/registries@2023-01-01-pr
 resource logAnalyticsWorkspace 'Microsoft.OperationalInsights/workspaces@2022-10-01' = {
   name: logAnalyticsWorkspaceName
   location: location
-  tags: {
```

Comment:
A really nice feature is that the `what-if` deploy accurately includes the tagging due to the Policy!